packages/sdk-cli/src/commands/build.ts

```diff
@@ -32,15 +45,7 @@ export const buildAction = async (cli: vorpal) => {
       { spaces: 2, EOL: os.EOL },
     );
   }
-
-  return new Promise((resolve) => {
-    const buildProcess = child_process.spawn(
-      'npm',
-      ['run-script', 'build'],
-      { stdio: 'inherit', shell: true },
-    );
-    buildProcess.on('exit', resolve);
-  });
+  return buildProcess();
```

Comment:
Should we use the rejected code and emit a log saying `Build failed with code ${code}`?